### PR TITLE
Added sample for https functions

### LINKED ISSUE #22: Pattern for Firebase function
### Description
Pattern on firebase functions:

#### If action
```
const actionNameExpress = ...

actionNameExpress .route('/', (req, res) => {});

exports.actionName = functions.https.onRequest(actionNameExpress)
```

#### if model (or database model)
```
const modelExpress = ...

modelExpress.get('/', (req, res) => {}); // getAll
modelExpress.get('/:id', (req, res) => {}); // getOne
modelExpress.post(/'', (req, res) => {}); // create
modelExpress.put('/:id', (req, res) => {}); // update
modelExpress.delete('/:id', (req, res) => {}); // delete

exports.modelName = functions.https.onRequest(modelExpress)
```

### PR DESCRIPTION
**Pull request by kuuuurt**
## Issue Reference
Fixes #22 
## Description
Added a sample for issue #22 

@tjmonsi

### CODE REVIEW

File: functions/https/actions.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const express = require('express');
```
Comment (tjmonsi) [inline functions/https/actions.js]: Why add express here if you already have baseExpress?

**Review by tjmonsi — COMMENTED** [review]
Would it be better to have a folder under `https` where you have an `actions` folder and `models` folder. Then each folder would have files like in `models` you have `user` and inside `user` you'll have `module.exports = function()` immediately?

File: functions/https/actions.js (hunk 1 above)
Comment (tjmonsi) [reply functions/https/actions.js]: @kuuuurt can you check?

Comment (kuuuurt) [reply functions/https/actions.js]: @tjmonsi it would be better. updated the code accordingly

**Review by tjmonsi — APPROVED** [review]

**Pull request merged**